orchestrator-salt/src/main/resources/salt-common/salt/fluent/init.sls

```diff
@@ -244,7 +280,7 @@ fluentd_delalyed_restart:
    cmd.run:
     - names:
        - "nohup sh /etc/td-agent/delayed_restart.sh > /etc/td-agent/delayed_restart.out 2>&1 &"
-       - "cp /etc/td-agent/td-agent_simple_profile.conf /etc/td-agent/td-agent.conf"
+       - "nohup sleep 30; cp /etc/td-agent/td-agent_simple_profile.conf /etc/td-agent/td-agent.conf &"
```

Comment:
Is 30 sec an empirical value? 😃

Reply:
it was added because at startup td-agent prints out the configuration then right after that re-reading it (so log did not tell something else is used),rearly it happens... probably few seconds wait would be enough before copying